### DISCUSSION TITLE
Should we upload superimages.zip everytime?

### DISCUSSION THREAD

**Discussion by ksauraj** [noobyysauraj/ofp_to_zip#7] [Q&A] [answered]
Should we upload superimages.zip everytime, cuz it takes too much time to get compressed and uploaded.

**Comment by ghost** [comment, marked as answer]
Frankly, `Yes` but also let user choose by providing a `simg2img` option in the inputs section. This way user can choose as per their requirements.

**Reply by ksauraj** [reply]
Yah that's a good idea.
